app/javascript/styles/mastodon/admin.scss

```diff
@@ -1054,6 +1054,10 @@ a.name-tag,
       }
     }
 
+    .icon {
+      padding-top: 5px;
+    }
```

Comment:
The reason for the misalignment is `vertical-align: top`, addressing it with a `padding-top` feels especially weird.

Reply:
Will look for alt method

Reply:
Updated to use `vertical-align: middle` in this context, which will override the `top` from higher up in same file.